src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -39,7 +39,7 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "owesMoney";
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_EMAIL = "This email already exists in the address book";
 
```

Comment:
Although the change seems basic, it requires a high level of attention to detail, so that minute changes like this can be implemented.